tests/cli/workflow/taxprofiler/test_cli_taxprofiler_config_case.py

```diff
@@ -0,0 +1,46 @@
+"""Tests cli methods to create the case config for Taxprofiler."""
+
+import logging
+from pathlib import Path
+from typing import List
+
+from _pytest.logging import LogCaptureFixture
+from click.testing import CliRunner
+
+from cg.cli.workflow.taxprofiler.base import config_case
+from cg.constants import EXIT_SUCCESS
+from cg.models.cg_config import CGConfig
+
+
+def test_without_options(cli_runner: CliRunner, taxprofiler_context: CGConfig):
+    """Test command without options."""
+    # GIVEN NO case_id
```

Comment:
There is a discrepancy between the  name and the comments. Are you testing without options or without case id. It would be great to be as specific as possible to know the important part of what is intended to be tested.

Is not case id an argument and not an option?

Reply:
I am testing with no options here. 

`cg workflow taxprofiler`

I will remove the comment so that there is a consistency.